README.md

```diff
@@ -25,29 +25,18 @@ Install this package as any other Python packages:
 $ pip install -U chaostoolkit-reporting
 ```
 
-Notice that this draws a few [dependencies][deps]:
+Notice that this draws a few [dependencies][deps] that will need to be installed:
 
 [deps]: https://github.com/chaostoolkit/chaostoolkit-reporting/blob/master/requirements.txt
 
+```
+$ pip install -U -r requirements.txt
+```
+
 Some of them are LGPL v3 licensed.
 
 If you are using Mac OS X then you will need to [install some additional dependencies](osx-install.md) that the `chaos report` command relies upon.
 
-You will also need to install the [pandoc][] package on your system.
-
-[pandoc]: https://pandoc.org/
-
-If you intend on creating PDF reports, the following additional packages will
-be needed:
-
-```
-$ sudo apt-get install texlive-latex-base \
-    texlive-fonts-recommended \
-    texlive-fonts-extra \
-    texlive-latex-extra \
-    pdflatex
-```
-
```

Comment:
(I don't know the answer to my question) - Is there a reason why you took this block out? Is it not needed?

Reply:
So `apt-get` is a Ubuntu (Debian and a few other linux OS') specific package manager, we could probably keep this block of text and instead say, `For Ubuntu you can run the following command to install the additional PDF packages:`

And then link to the `osx-install.md` file for people who want to do this on Mac 😄

Reply:
I agree, I was surprised this was removed.

Mind you, this will be removed the day we implement this issue: https://github.com/chaostoolkit/chaostoolkit-reporting/issues/25

Reply:
Would it be best to contain that information within the osx-install.md file as well then along with the Mac installation alternative or just leave it where it is and give it an appropriate header?

Reply:
It's fine by me to have a separate document for each platform, or a single one with all platforms.

Reply:
I'll try and adapt the osx-install.md file then to contain alternatives for each platform and let you know what I come up with